data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,470 @@ mission "Unfettered returning home"
+			`	You manage to land a shot, although it seems to do nothing to the large spacecraft before you. Then you hear a small explosion. You aren't sure whether the damaged ship's systems failed on their own, or if by some stroke of luck, your shot did something to it.`
+			`	Unfortunately, you do not get a chance to ponder it, as the ship abruptly explodes, taking you and everyone else on the beach down with it.`
+				die
+			label miss
+			`	You miss your shot. Neither the ship nor the two Hai react. You feel like you heard a chuckle, but you're not sure where it came from.`
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+		log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+
+
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage Reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>."
+	clearance
+	deadline 7
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`Upon returning to the spaceport, a worker immediately greets you while others begin to fill your ship with crates. A quick check of the shipment data shows that the cargo has already been allocated in your name. All that's left is to deliver it to <destination> by <date> for a promised sum of <payment>.`
+			`	As you head to the bridge, preparing to take off from the planet, you notice a message on the command panel:`
+			`	"You would be wise to use the Anomalocaris as your flagship."`
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides.`
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are clearly not happy when they somehow find out you didn't deliver the cargo in time. You receive a short message from them: "You will not prove yourself by failing our requests."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Anomalocaris, or some of the cargo was on another ship.`
+	to complete
+		has "ship model: Anomalocaris"
+	on complete
+		dialog `You receive a message from the associates you bought the ship from: "Thank you for helping our brethren, <last>. We will not forget this. Please accept these <payment>."`
```

Comment:
```suggestion
		dialog `After landing, a group of Unfettered gather around your ship and begin unloading the medical supplies, though strangely they don't seem to be in any hurry. Shortly after they've finished, you receive a rather brief message from the Unfettered you bought the ship from. "Thank you for helping our brethren, <last>. Please accept these <payment>."`
```
A few things here:

Since this comes before HL 3 below, it makes sense that this is where your ship landing is talked about, not that you've gotten a message and paid, and then "as you land" occurs after the fact below. (I'll get to that in the next review chunk.)

It seems a little odd that they say they "won't forget this" in a quick manner when this mission is in actuality them setting you up to bring that ship to their world. Along with that, I made note that the message was rather brief for two who were very involved and suddenly not as important anymore (and somewhat shady since the mission is in actuality them using you to smuggle the ship. This opens up some interesting dialog options for HL3.

Above there, you also refer to them as the "Unfettered and their associates", so it's probably best to refer to the two you met as just "Unfettered".